src/lsdb/catalog/catalog.py

```diff
@@ -1,10 +1,20 @@
-from typing import Dict
+from __future__ import annotations
```

Comment:
Opened https://github.com/astronomy-commons/lsdb/issues/25 to take out these kinds of lines.